app/models/user.rb

```diff
@@ -56,4 +56,10 @@ def kind
 
     "normal"
   end
+
+  def reinvitable?
+    return true if invitation_status == "invited" && invitation_sent_at <= 7.days.ago
```

Comment:
This should probably be `... invitation_sent_at > 7.days.ago` shouldn't it?

Reply:
No. You're looking at it backwards. The invitation_sent_at "date" should be less than or equal to whatever date was 7.days.ago. 

In other words. 7.days.ago should be closer to now than invitation date.

Reply:
:man_facepalming: 

yes. :)